week-4/F-object-keys/exercise-part-2.js

```diff
@@ -33,14 +33,14 @@ var storeBranches = {
 
 // # 1
 // prints [ 'glasgow', 'edinburgh' ]
-console.log();
+console.log(Object.keys(storeBranches));
 
 // # 2
 // prints [ 'manager', 'assistant', 'interns' ]
-console.log();
+console.log(Object.keys(storeBranches.glasgow));
 
 // # 3
 // prints [ 'head_intern', 'intern' ]
-console.log();
+console.log(Object.keys(storeBranches.edinburgh.interns));
```

Comment:
And all the Object keys stuff is correct. Looks like the week-4 stuff didn't pose you too many problems.